ginivision/src/main/java/net/gini/android/vision/internal/camera/api/SizeSelectionHelper.java

```diff
@@ -0,0 +1,61 @@
+package net.gini.android.vision.internal.camera.api;
+
+import android.hardware.Camera;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import net.gini.android.vision.internal.camera.photo.Size;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @exclude 
+ */
+public final class SizeSelectionHelper {
+
+    @Nullable
+    public static Size getLargestSize(@NonNull List<Camera.Size> sizes) {
```

Comment:
Because `new Size(0, 0)` for initializing the preview and picture sizes in the `CameraController`. We could also work with `Camera.Size`, but since it's constructor is private, it's good I think, if we have our own class, even though it does nothing special.